src/mergeForEach.ts

```diff
@@ -14,7 +14,7 @@ export type ComparisonResult = number
  * Can be used to implement efficient algorithms which profit from sorted data
  * like `mergeSort` or `mergeJoinWith`.
  *
- * Both collections must be sortable. They will be sorted ascendently using
+ * Both collections must be sortable. They will be sorted ascendantly using
```

Comment:
why the change? both are good

Reply:
When I copied it to my comment in Github, Chrome's autocorrector complained about it 😆